src/components/calculator.js

```diff
@@ -2,22 +2,15 @@ import React, { useState } from 'react';
 import PropTypes from 'prop-types';
 import calculate from '../logic/caculate';
 
-class Buttons extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {};
-    this.clickEvent = this.clickEvent.bind(this);
-  }
-
-  clickEvent() {
-    const { clickEvent, btnName } = this.props;
+function Buttons(props) {
+  const { className, btnName } = props;
+
+  const clickEvent = (props) => {
+    const { clickEvent, btnName } = props;
```

Comment:
- [ ] The app buttons don't work anymore. As a result, I'm unable to make any mathematical computation with your calculator app. Here's an image of the console error 👇 
![image](https://user-images.githubusercontent.com/46839548/173182604-4b918960-efb6-41f5-a777-8dc60c410559.png)

Reply:
Fixed.